### ISSUE TITLE
edit to WCAG section

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-quick-start#82]
<h>WCAG SC</h>
Web Content Accessibility Guidelines (WCAG) is the international standard for making web content more accessible to people with disabilities. The WCAG requirements are called "success criteria" (SC). Learn more from the <a href="/WAI/intro/wcag">WCAG Overview</a>.

**Comment by slhenry** [comment]
Thanks.
I'm still not sure about "What is WCAG?" as a heading. Maybe "About WCAG" or just "WCAG"? Maybe it's worth getting input from others?
(minor: Is the comma missing at the end on purpose or a typo?)

**Comment by iadawn** [comment]
Not had any comments from anyone else on this, I am happy for people to yeah or nay it but don't know how much that is necessary.

I put in 'About WCAG' as I think 'WCAG' on it's own is a bit abrupt and unclear.

I purposefully omitted the comma at the end, but unintentionally omitted the full stop 😉

**Comment by slhenry** [comment]
Re: "Not had any comments from anyone else on this" -- that's because we didn't flag it as a question so people didn't know to comment on it. :-)

I'm fine with "About WCAG" and if you are, then I don't think we need to get more feedback. However, we probably want to check that @nitedog is now OK with this box - location, heading, text, visual design, etc. -- and then point EOWG to it for review. (just a minor question)

**Comment by iadawn** [comment]
> that's because we didn't flag it as a question so people didn't know to comment on it. :-)

My experience so far is that that seldom stops anyone 😉

I will flag it to Shadi specifically in case he misses this GitHub email.

**Comment by nitedog** [comment]
Looks great to me! Just remove the second <abbr> around WCAG towards the end of the paragraph (we already have the expansion in the first sentence). Maybe also from the heading? - it is explained in the sentence right below, and the phrase "about WCAG" suggests that the text will explain the acronym.

**Comment by iadawn** [comment]
Sorry, when you say

> remove the second around WCAG

I am not sure you are referring to. I might guess abbr element, but I wanted to check

**Comment by slhenry** [comment]
+1 to removing the abbr

**Comment by iadawn** [comment]
-1 to removing:

[Sufficient techniques](http://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html#meaning-located-techniques-head) indicate that abbreviations should be defined on first encounter and defined in abbr on each subsequent encounter. I appreciate these are sufficient techniques, but these are things that cause problems for people with cognitive disabilities and memory problems.

**Comment by yatil** [comment]
[Just a thought] For the heading, what about “About Web Accessibility” – It would remove the use of WCAG there and give a broader entrance to the topic. WCAG could then be introduced in the 1st sentence. 

I don’t feel strongly about removing the abbr, but we wouldn’t use them every time when we write HTML as well.

**Comment by iadawn** [comment]
Actually, as someone who has an exceedingly weird and bad memory, I would be happier if people did use them **every** time acronyms were used. The only reason we wouldn't consider it for HTML is because that is a common term in our domain of knowledge, in the same way the Rx is common in medical and ACMP is be common in marine biology.

**Comment by yatil** [comment]
Probably it makes more sense to leave it in here as this is for starters :-)

**Comment by nitedog** [comment]
The reason why it is not good practice to use abbreviations every time, is because they are read aloud every time. I don't think abbreviations (with title attribute) need to be coded three times within the same paragraph when they are expanded earlier on in that same section. I also disagree with your analysis of WCAG, as it suggests several approaches and not necessarily using abbreviations as the only mechanism and so wildly throughout. I think this is a severe overuse that may not help people with cognitive disabilities the way you think it may be helping them - for example people with dyslexia using screen readers and getting that read aloud so often may be more distracting than helpful.

**Comment by iadawn** [comment]
I have removed the heading and last abbr element.
